docs/v2.0/README.md

```diff
@@ -0,0 +1,327 @@
+# Instamojo PHP API [![Latest Stable Version](https://poser.pugx.org/instamojo/instamojo-php/v/stable)](https://packagist.org/packages/instamojo/instamojo-php) [![License](https://poser.pugx.org/instamojo/instamojo-php/license)](https://opensource.org/licenses/MIT)
```

Comment:
we just need one readme. Users just need to see the documentation for the selected version of wrapper.